src/Slider/Slider.jsx

```diff
@@ -401,9 +404,22 @@ const Slider = class Slider extends React.Component {
     });
   }
 
+
   play() {
-    const { playDirection } = this.props;
-    this.interval = setInterval(playDirection === 'forward' ? this.playForward : this.playBackward, this.props.interval);
+    if (this.props.intervalList) {
+      let priorIntervals = 0;
+      for (let i = 0; i < this.props.intervalList.length - 1; i += 1) {
+        setTimeout(this.props.playDirection === 'forward' ? this.playForward : this.playBackward, this.props.intervalList[i] + priorIntervals);
```

Comment:
Is this setting a timeout for each interval listed in the array?

Reply:
What it is doing is making the slides animate individually after a set amount of time that includes the sum of time waited for each prior slide.